benchmarks/benchmark_runner.hpp

```diff
@@ -283,46 +283,70 @@ struct BenchmarkRunnerGemm {
       state.SkipWithError("Disposition Failed.");
     }
 
-    auto tflop = ((2.0 * options.m * options.n * options.k * options.l) * 1e-12);
-    auto giga_bytes_transferred = (((options.m * options.k) * sizeof(ElementA) +
-                                     (options.k * options.n) * sizeof(ElementB) +
-                                 (options.beta != 0 ? 2 : 1) * (options.m * options.n) * sizeof(ElementC)) * 1e-9) *
-                                 options.l;
+    state.counters["m"] = options.m;
+    state.counters["n"] = options.n;
+    state.counters["k"] = options.k;
+    state.counters["l"] = options.l;
+    state.counters["alpha"] = options.alpha;
+    state.counters["beta"] = options.beta;
+
+    std::stringstream extra_label;
+    if constexpr (cute::size<0>(StrideA{}) == 1) {
+      extra_label << "layoutA=ColumnMajor ";
+    } else if constexpr (cute::size<1>(StrideA{}) == 1) {
+      extra_label << "layoutA=RowMajor ";
+    }
+    if constexpr (cute::size<0>(StrideB{}) == 1) {
+      extra_label << "layoutB=RowMajor ";
+    } else if constexpr (cute::size<1>(StrideB{}) == 1) {
+      extra_label << "layoutB=ColumnMajor ";
+    }
+    if constexpr (cute::size<0>(StrideC{}) == 1) {
+      extra_label << "layoutC=ColumnMajor ";
+    } else if constexpr (cute::size<1>(StrideC{}) == 1) {
+      extra_label << "layoutC=RowMajor ";
+    }
+    state.SetLabel(extra_label.str());
+
+    auto gflop = 2.0 * options.m * options.n * options.k * options.l * 1e-9;
+    auto mega_bytes_transferred = static_cast<double>(
+        options.m * options.k * sizeof(ElementA) +
+        options.k * options.n * sizeof(ElementB) +
+        (options.beta != 0 ? 2 : 1) * options.m * options.n * sizeof(ElementC)
+      ) * 1e-6 * options.l;
```

Comment:
why are we measuring MBPS, when we would typically get the figure in the Gigabytes range